src/main/scala/xiangshan/frontend/icache/ICacheCtrlUnit.scala

```diff
@@ -0,0 +1,293 @@
+/***************************************************************************************
+ * Copyright (c) 2020-2021 Institute of Computing Technology, Chinese Academy of Sciences
+ * Copyright (c) 2020-2021 Peng Cheng Laboratory
```

Comment:
maybe update it to 2024 and add "2024 bosc"

Reply:
Just copy-pasted from ICacheMainPipe.scala, does that need update too?

Btw, is this correct?
https://github.com/OpenXiangShan/XiangShan/blob/72dab9745cef33532f03ca252d63455645aaab8a/src/main/scala/xiangshan/frontend/icache/ICache.scala#L2-L4

Reply:
AFAIK is correct